From 75d7ab8f154d283b7b3101cb0389e53bd6bc8c90 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:09:03 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 from keyservers/WKD:
same certificate(s), with self-signatures our copy was missing; the
commands are in the spec comment.

Assisted-by: Claude Opus 5.5
---
 A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 | 100 +++++++++++++++--------
 weechat.spec                             |   7 +-
 2 files changed, 70 insertions(+), 37 deletions(-)

diff --git a/A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 b/A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8
index 3302ef4..f3a2ac2 100644
--- a/A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8
+++ b/A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8
@@ -12,40 +12,68 @@ dN4KoaULR2fGu+7NHeZBRk0r7+OjtRaXZWyaYfx4XAKqqdxcDcgYS8vb111JpkFN
 /zf2HZtIA7IMUGQrf23O3xUun6voQb14qEMtWS49DxXmBc5g2KITtkpoVBSzHNVO
 6aUzpgjeV8bRGTobpVvilz6Af34/gYIsvo0JQ14xB/F7cyaA1e4d6BtzSwARAQAB
 zS1XZWVDaGF0IChzaWduaW5nIGtleSkgPHdlYm1hc3RlckB3ZWVjaGF0Lm9yZz7C
-wXgEEwECACIFAlCCu4cCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJEPgv
-SxbexAj4JAoP+gMurI1HgmWK7pqO+/Hn5SAMKvHYwtHTpGqdDfponnTW3JgWUm2X
-fgvHk5ZAew6a6hbLE73v8LSbepbCM6mzdtqO4FPXV8t+x/qLu3zbi9Q6Fd+bP+2G
-ZDfRyZdk1UaeW2wOhwcvP8+kQxBB8gK7kgaD7rWG4CbU1jTQjdpBsibT/iXhszaC
-kIe0ExaUYS7B58IbumjH+B3tcwhOW0SKImY5H4DKYAgWQ5/oSOwGf3fp3K2xT05c
-MBiywTBBlXmtSY77yA0YxTYi/vIkPOuqKhMwc9Si5Ak4U2NvInGa3WOQoXH5OWYr
-Ipe4nzpFMcBt0tMDbErkQsQFVsZE9YrOdoAHvMqn3Dxo99N3vX+CxmziWpzF2V6c
-RyqFEFc0IYxABLpKNTgL9WUy+dwMsvuGR6/sob9hE5drJ8YeDEWLGZuVMys4vTvE
-ctP+cedxgz2ZTyvWPCaR7k7L5yOvBTIwvYG6pu7utpxVnIpVRDs3hXl3VPNbu4SC
-0JpAC0fB77VlOCbI205GTbcFxh82qiyJSZRmZF+tXOlBna7Jq/Fmtdyfh6COi5Tw
-EVKzdXc+uBNxmrgkf1AIWaKXP5IdvHs75c+/WW93bjTC2QmUtv3NZG4HPo2NX6G+
-QMkMJOO+vnAQzYo/2BM6s2zCnR8oFKPt3t9ktz992D3PDkYwClKsz+8BzsFNBFCC
-u4cBEADWYSdT6uWblZ4T/kiOjwdklcIuNCJhII4bXNlyLjm2xT2AescwQz1MVGO+
-UVIXk6YKYwT3I+CRx/UutTTD8s03yE3Lc5scztdp/S1pEX1ahvVI3elj0+DpQvmo
-QB5aOL555LskFDo8dUHng/q17IHonHuGnIK5ufLD/CMvSzIOmL/5PBKyI3TZ21N5
-pz+4PzG0wSMTSQctiSEpccefuXyqBDfaxJVuuE3y/8AZy2hbp6EHjJkGiHE89yXx
-rpJkPUv9i8D7M98uoERon6ma7xa/YHh3QROSG+oX8AuExmhbSr9ZkgTbfcehQoZE
-pAgTqsxBhPirBnOr1lDAdHv7Zjr2qPG2uezAo1mkC7q6qQqfDJQHYTTOBDsin/TJ
-pKma2ZLvY775otAS0nwxwvQAA4LWtJntlMZAZPegrVfre3tzM5V9MSeGME1oFv6L
-9MrlpTJbym1kOa4agV+e1/ADEDPHERunyc735PrT1oTYbH5wQpTXPRdgUIFK//++
-6fZy7hO9JW/xdwPTf4FROL4T1tCg/hW/MC7+/rSQZ50Ucf4hgR1QVjudMMPDVqqH
-C+9EKE1/pIVn6FNGN0YfPl3/NeV0daMceOZh1+HdBfAcPWuOUOIpUkAt7gAO7hoI
-EVMaVFOzdTTufFzohV0volqsAyBHID77h/mjRTgCcynNZryzsQARAQABwsFfBBgB
-AgAJBQJQgruHAhsMAAoJEPgvSxbexAj42WYQAIq5zKYsppAy1DeWL8ltmObEjSs/
-j07Er62KBb3jzbzGbjQybcneb29I7knmugPVo/QEnUzwOQHRbzMZ43pkcSiPNmhA
-8NMgbldrmzhWC/5ueFwC4zXBEFhAuVwCyX6AlXrGyrxM72ubuCTziVjKB4LtIFfA
-EsKRUF2r14U3mcJuSm/+rvJBEdOc0ccRHQznkARc945KOAD/FY1cXHP4Fo7SfT8w
-KCGQGn445Lop7rHKhrhZ+6eenPIAcoVUEK1aef57VItcSmT3IAMbrkMZaCX8MEQW
-GMdBt/gOfPmkmmGwEvFqkyBoerHYmgaB4SOFwjWgrgMBiuihUGZZfdJviF+NB687
-67/ikvot/ACSOzsB2BslNh6jGPwOUdRE4b/B1CtDm7Ukb+w1RbumQCtO1M64mh9w
-bPp8bnjvS5OW1E0gEqZFoXNJPYUV35IubWeOL2+LPltJE5p9nzTtvyXG2aJhlnW5
-SpN7b/OD0wSu0STxdbmUTJ1WFz7P8HhmT1d5cRS8Q/5IjwWJRAyoyY/CBEYjz+Fs
-eFF2jvxWceJHsTo7KhKLVAcB5tAqAdbpkbEF65FDzsFQz5Ql+C3TiQaNHRrHyrUl
-g/NqrZmPiNTvjp3Mh5yj5e8mkOR/96YuHjF72YWnybVCt5+olFrr9v0+uCJU168O
-bZlCOWLXBLOqK0Z0
-=MySb
+wdIEEwEIAIYFgmeNAuQECwkIBwkQ+C9LFt7ECPhHFAAAAAAAHgAgc2FsdEBub3Rh
+dGlvbnMuc2VxdW9pYS1wZ3Aub3JnzGQ3tc/PT4h8ebRW6b2HhNoh4zlUpUbTpOAJ
+rDa1ig8DFQgKBBYCAwECF4ACGwMCHgEWIQSpq1q3ePpcNSL9A3j4L0sW3sQI+AAA
+5csP/2h8uzNT0xkP6vYlPBwHL4ilBdDjqhHp+Xw41DqU0+J7YzpdNUGxzAPR0Urj
++KjCakfw8b5FhgiT5r3KIa+TziRKjwWydjoB7cvZntJk7md5D6rI1/dvyKQONreR
+Av5ORS7X1cVsCK5w2svmVmRwqvJoBrwd1nX0Qa52MGytb4Un/SDTkCiyVg2VI/ZZ
+RTnA6gLyrIDDp9sDLJ6jB/oF9cs9HEjejX/q4gYVvGaipHq9eHizfH2kcNLH3yKZ
+G4ekb/cA+E3HayRTmHqmuRv0FexTqQAx+Zr6NtS+OUM2C+yGy6Yg6P32g9mgUJyE
+zpicYWsuRrpPG105pLr4rZl60cxwKnw+O5+KOAjdYzrX7r/qVviMw0qOEc+bN724
+eOQ9g6tWQv0Vt4r5wDxQfLdRY0+zQKa1Ryb6eXOuYUlDOeOk9xkGhGT4qS9BjJoV
+NdxKStuEHhPyjL8bCfxvDBXrxH7vqjZF8xZmYJfDP6tjF1iHJYV4Ux8NJ1mvNKUm
+okOytgaVDCZfRWMmhF0RHz1wq+XrCIJG9LeJun6ycer/qHnfTCKezNlxMi+/fz0g
+UNlZs8fIYU2e8aTw1ms+3lYiRDOiMHDlAeKydXZtsBV2XrylNQdl/64jRilgfU9B
+6iTEJQkvUveo01UgRwJBSNz+lwJy+mQA22eRQkjyXQtmH+ITwsGPBBMBAgAiBQJQ
+gruHAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAhCRD4L0sW3sQI+BYhBKmr
+Wrd4+lw1Iv0DePgvSxbexAj4JAoP+gMurI1HgmWK7pqO+/Hn5SAMKvHYwtHTpGqd
+DfponnTW3JgWUm2XfgvHk5ZAew6a6hbLE73v8LSbepbCM6mzdtqO4FPXV8t+x/qL
+u3zbi9Q6Fd+bP+2GZDfRyZdk1UaeW2wOhwcvP8+kQxBB8gK7kgaD7rWG4CbU1jTQ
+jdpBsibT/iXhszaCkIe0ExaUYS7B58IbumjH+B3tcwhOW0SKImY5H4DKYAgWQ5/o
+SOwGf3fp3K2xT05cMBiywTBBlXmtSY77yA0YxTYi/vIkPOuqKhMwc9Si5Ak4U2Nv
+InGa3WOQoXH5OWYrIpe4nzpFMcBt0tMDbErkQsQFVsZE9YrOdoAHvMqn3Dxo99N3
+vX+CxmziWpzF2V6cRyqFEFc0IYxABLpKNTgL9WUy+dwMsvuGR6/sob9hE5drJ8Ye
+DEWLGZuVMys4vTvEctP+cedxgz2ZTyvWPCaR7k7L5yOvBTIwvYG6pu7utpxVnIpV
+RDs3hXl3VPNbu4SC0JpAC0fB77VlOCbI205GTbcFxh82qiyJSZRmZF+tXOlBna7J
+q/Fmtdyfh6COi5TwEVKzdXc+uBNxmrgkf1AIWaKXP5IdvHs75c+/WW93bjTC2QmU
+tv3NZG4HPo2NX6G+QMkMJOO+vnAQzYo/2BM6s2zCnR8oFKPt3t9ktz992D3PDkYw
+ClKsz+8BzsFNBFCCu4cBEADWYSdT6uWblZ4T/kiOjwdklcIuNCJhII4bXNlyLjm2
+xT2AescwQz1MVGO+UVIXk6YKYwT3I+CRx/UutTTD8s03yE3Lc5scztdp/S1pEX1a
+hvVI3elj0+DpQvmoQB5aOL555LskFDo8dUHng/q17IHonHuGnIK5ufLD/CMvSzIO
+mL/5PBKyI3TZ21N5pz+4PzG0wSMTSQctiSEpccefuXyqBDfaxJVuuE3y/8AZy2hb
+p6EHjJkGiHE89yXxrpJkPUv9i8D7M98uoERon6ma7xa/YHh3QROSG+oX8AuExmhb
+Sr9ZkgTbfcehQoZEpAgTqsxBhPirBnOr1lDAdHv7Zjr2qPG2uezAo1mkC7q6qQqf
+DJQHYTTOBDsin/TJpKma2ZLvY775otAS0nwxwvQAA4LWtJntlMZAZPegrVfre3tz
+M5V9MSeGME1oFv6L9MrlpTJbym1kOa4agV+e1/ADEDPHERunyc735PrT1oTYbH5w
+QpTXPRdgUIFK//++6fZy7hO9JW/xdwPTf4FROL4T1tCg/hW/MC7+/rSQZ50Ucf4h
+gR1QVjudMMPDVqqHC+9EKE1/pIVn6FNGN0YfPl3/NeV0daMceOZh1+HdBfAcPWuO
+UOIpUkAt7gAO7hoIEVMaVFOzdTTufFzohV0volqsAyBHID77h/mjRTgCcynNZryz
+sQARAQABwsG+BBgBCAByBYJnjQLkCRD4L0sW3sQI+EcUAAAAAAAeACBzYWx0QG5v
+dGF0aW9ucy5zZXF1b2lhLXBncC5vcmf/QZdg5pWknEK1cSN1v65kVyVps6H7/EkN
+gTjmcCxz0wIbDBYhBKmrWrd4+lw1Iv0DePgvSxbexAj4AAA2xRAAsYrEKjsQU91N
+RW5u3EGGudUYVEdAXvNl7WVmDiUe/tyt01PbFxeplKtEQ87j4xvWRnHQl/J0Aium
+a023iwj/hXLru3QSy5u5hlge13Eh3K00GhEijHl80Hhs2VYDGAFTaZdqQtyBRf6R
+QDZTGwvfaYdyqKuCa6xUQzYHPZmT6gmgeDg73mrWkS1Z4y2A/9zXPwcPCmwmFZqc
+bMfKSLjOk7WDMKUZ7ObIqJ75fYqCJ81gPSEFMUzAZOmxoselnZjoenU5tr09/pTw
+7t4XEFxMaLXLsS0srU72Ljj+E163iS696u3zAeqxF02e0OzBosAKc6PTGIPy2NiG
+M5Ivi8LW/TCSEnZWBncsQVFX8X6WiTvm32FoMxb88Y0UgouWsOKaW36qoiMkwJeD
+2gNyL7nLpi+sdHOJs1oFHsA7M3tUXXRIk9ZEOdmP2FCbfYFpJIotYLhDGWYv/JmY
+E4V08bplfmVzXD70mMisAUHCd3mvv5oCFshn3YgwMuR3zuQsPmi6OVwWcMLdWaCp
+U4habVjUqdzSvaN9VuxfTFMASflZNlICA8zB2vTX/8IpvBAH9aEgInKc7hEjRdtp
+CBiMny4fn9D3neqjqR4P88GOokjzYIfyP92bZuA0nHIOyJ+VeSQoPdVP4gTJ2m1/
+mT7bzQhUCzY7xUhneS6GEKkznPQOyS/CwXYEGAECAAkFAlCCu4cCGwwAIQkQ+C9L
+Ft7ECPgWIQSpq1q3ePpcNSL9A3j4L0sW3sQI+NlmEACKucymLKaQMtQ3li/JbZjm
+xI0rP49OxK+tigW94828xm40Mm3J3m9vSO5J5roD1aP0BJ1M8DkB0W8zGeN6ZHEo
+jzZoQPDTIG5Xa5s4Vgv+bnhcAuM1wRBYQLlcAsl+gJV6xsq8TO9rm7gk84lYygeC
+7SBXwBLCkVBdq9eFN5nCbkpv/q7yQRHTnNHHER0M55AEXPeOSjgA/xWNXFxz+BaO
+0n0/MCghkBp+OOS6Ke6xyoa4WfunnpzyAHKFVBCtWnn+e1SLXEpk9yADG65DGWgl
+/DBEFhjHQbf4Dnz5pJphsBLxapMgaHqx2JoGgeEjhcI1oK4DAYrooVBmWX3Sb4hf
+jQevO+u/4pL6LfwAkjs7AdgbJTYeoxj8DlHUROG/wdQrQ5u1JG/sNUW7pkArTtTO
+uJofcGz6fG5470uTltRNIBKmRaFzST2FFd+SLm1nji9viz5bSROafZ807b8lxtmi
+YZZ1uUqTe2/zg9MErtEk8XW5lEydVhc+z/B4Zk9XeXEUvEP+SI8FiUQMqMmPwgRG
+I8/hbHhRdo78VnHiR7E6OyoSi1QHAebQKgHW6ZGxBeuRQ87BUM+UJfgt04kGjR0a
+x8q1JYPzaq2Zj4jU746dzIeco+XvJpDkf/emLh4xe9mFp8m1QrefqJRa6/b9Prgi
+VNevDm2ZQjli1wSzqitGdA==
+=LVC2
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/weechat.spec b/weechat.spec
index 24c9868..3a34c5c 100644
--- a/weechat.spec
+++ b/weechat.spec
@@ -10,6 +10,10 @@ License:   GPL-3.0-only
 
 Source:    https://weechat.org/files/src/%{name}-%{version}.tar.xz
 Source1:   https://weechat.org/files/src/%{name}-%{version}.tar.xz.asc
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 | sq keyring filter --experimental --cert A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 --output update.pgp
+#   sq keyring merge --output A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8.new A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8 update.pgp
+#   mv A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8.new A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8
 Source2:   https://keys.openpgp.org/vks/v1/by-fingerprint/A9AB5AB778FA5C3522FD0378F82F4B16DEC408F8
 
 BuildRequires: asciidoctor
@@ -44,6 +48,7 @@ BuildRequires: tcl-devel
 BuildRequires: uriparser-devel
 %endif
 BuildRequires: zlib-devel
+BuildRequires: openpgpverify
 
 Requires:      hicolor-icon-theme
 
@@ -72,7 +77,7 @@ Requires: %{name}%{?_isa} = %{version}-%{release}
 This package provides the PHP plugin for weechat.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup -p1 -n %{name}-%{version}
 
 
-- 
2.56.0

